samples/grid-lite/demo/grid-cell-format/demo.html

```diff
@@ -1,4 +1,4 @@
-<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/grid/grid-lite.js"></script>
 
 <div id="container"></div>
```

Comment:
Just a suggestion to add a theme class to the container to be able to get the background variable from it (for dark mode).
```suggestion
<div id="container" class="hcg-theme-default"></div>
```